src/app/tests/suites/DL_UsersAndCredentials.yaml

```diff
@@ -1274,7 +1274,7 @@ tests:
               - name: "operationType"
```

Comment:
Shouldn't this enable the "Disabled because due to https://github.com/project-chip/connectedhomeip/issues/21656" test above?